src/config/config.ts

```diff
@@ -9,7 +10,9 @@ import { Locale } from '../i18n';
  *  For multi-level features, only the top-level featureName should be
  *  toggled.
  */
-export interface SandboxFeatures {}
+export interface SandboxFeatures {
+  Fintraffic: boolean;
```

Comment:
It would be nice to document each sandbox feature with a comment explaining what it is